dataland-rabbitmq/rabbitmq.conf

```diff
@@ -1,2 +1,2 @@
 management.load_definitions = /etc/rabbitmq/definitions.json
-disk_free_limit.relative=1.5
+disk_free_limit.absolute = 768MB
```

Comment:
From the Jira-Ticket: "512 MB RAM and 1,5*512MB for disk space and add a comment to change either one if the other one is changed" I do not see a comment like that anywhere

Reply:
Sorry. Comments have been added and pushed now.